terraform/projects/web-platform-tests-live/compute.tf

```diff
@@ -0,0 +1,35 @@
+resource "aws_instance" "production" {
+  ami = "${data.aws_ami.ubuntu.id}"
+  instance_type = "t2.small"
+  key_name = "${var.key_name}"
+  subnet_id = "${element(module.subnet.ids, 0)}"
+  vpc_security_group_ids = [
+    "${aws_security_group.web_platform_test_live.id}",
+  ]
+  tags {
+    "Name" = "${var.name}-production"
+  }
+}
+
+resource "aws_eip" "production" {
+  instance = "${aws_instance.production.id}"
+  vpc      = true
+}
```

Comment:
I see this is where the elastic ip is reserved. I assume the DNS assignment happens outside of this terraform repo?